articles/active-directory/saas-apps/lessonly-tutorial.md

```diff
@@ -68,51 +68,51 @@ To configure the integration of Lesson.ly into Azure AD, you need to add Lesson.
+For single sign-on to work, Azure AD needs to know what the counterpart user in Lessonly.com is to a user in Azure AD. In other words, a link relationship between an Azure AD user and the related user in Lessonly.com needs to be established.
 
-In Lesson.ly, assign the value of the **user name** in Azure AD as the value of the **Username** to establish the link relationship.
+In Lessonly.com, assign the value of the **user name** in Azure AD as the value of the **Username** to establish the link relationship.
 
-To configure and test Azure AD single sign-on with Lesson.ly, you need to complete the following building blocks:
+To configure and test Azure AD single sign-on with Lessonly.com, you need to complete the following building blocks:
 
 1. **[Configuring Azure AD Single Sign-On](#configuring-azure-ad-single-sign-on)** - to enable your users to use this feature.
 1. **[Creating an Azure AD test user](#creating-an-azure-ad-test-user)** - to test Azure AD single sign-on with Britta Simon.
-1. **[Creating a Lesson.ly test user](#creating-a-lessonly-test-user)** - to have a counterpart of Britta Simon in Lesson.ly that is linked to the Azure AD representation of user.
+1. **[Creating a Lessonly.com test user](#creating-a-lessonly-test-user)** - to have a counterpart of Britta Simon in Lessonly.com that is linked to the Azure AD representation of user.
 1. **[Assigning the Azure AD test user](#assigning-the-azure-ad-test-user)** - to enable Britta Simon to use Azure AD single sign-on.
 1. **[Testing Single Sign-On](#testing-single-sign-on)** - to verify whether the configuration works.
 
 ### Configuring Azure AD single sign-on
 
-In this section, you enable Azure AD single sign-on in the Azure portal and configure single sign-on in your Lesson.ly application.
+In this section, you enable Azure AD single sign-on in the Azure portal and configure single sign-on in your Lessonly.com application.
 
-**To configure Azure AD single sign-on with Lesson.ly, perform the following steps:**
+**To configure Azure AD single sign-on with Lessonly.com, perform the following steps:**
 
-1. In the Azure portal, on the **Lesson.ly** application integration page, click **Single sign-on**.
+1. In the Azure portal, on the **Lessonly.com** application integration page, click **Single sign-on**.
 
 	![Configure Single Sign-On][4]
 
 1. On the **Single sign-on** dialog, select **Mode** as	**SAML-based Sign-on** to enable single sign-on.
  
 	![Configure Single Sign-On](./media/lessonly-tutorial/tutorial_lesson.ly_samlbase.png)
 
-1. On the **Lesson.ly Domain and URLs** section, perform the following steps:
+1. On the **Lessonly.com Domain and URLs** section, perform the following steps:
 
 	![Configure Single Sign-On](./media/lessonly-tutorial/tutorial_lesson.ly_url.png)
 
     a. In the **Sign-on URL** textbox, type a URL using the following pattern:
 	| |
 	|--|
-	| `https://<companyname>.lesson.ly/signin`|
+	| `https://<companyname>.lessonly.com/signin`|
```

Comment:
At this point, we're displaying the same url twice - do we want that @susanesho?

Reply:
thanks looks like my latest changes were not updated when I pushed, thanks for noting it.